tbot_contrib/gpio.py

```diff
@@ -24,20 +41,77 @@ def set_direction(self, direction: str) -> None:
         self._direction = direction
 
     def set_active_low(self, value: bool) -> None:
+        """Set Active Low
+
+        When Active Low is ``True``, the GPIO triggers on a ``LOW`` singnal, else on a ``HIGH`` signal
+
+        Args:
+            value (bool): Value
+        """
         (self._gpio_path / "active_low").write_text("1" if value else "0")
 
     def get_active_low(self) -> bool:
+        """Get Active Low
+
+        Reads value and returns direction ``in`` or ``out``
+
+        Returns:
+            bool: Value
```

Comment:
And similarly, return values are documented like this:

```suggestion
        :rtype: bool
        :returns: Value
```